JustBrewIt/MyBreweryAPI.cs

```diff
@@ -77,14 +77,14 @@ public enum Country { UnitedStates };
 
     public enum State { Ohio };
 
-    public partial class Welcome
+    public partial class BreweryAPI
     {
-        public static Welcome[] FromJson(string json) => JsonConvert.DeserializeObject<Welcome[]>(json, QuickType.Converter.Settings);
+        public static BreweryAPI[] FromJson(string json) => JsonConvert.DeserializeObject<BreweryAPI[]>(json, BreweryQuickType.Converter.Settings);
```

Comment:
Is it a BreweryAPI, or a Brewery?

BreweryAPI == service class that does something.  A verb.
Brewery: a noun, what a service class will return.

This feels more like a noun.

Reply:
Thank you prof for the feedback.  I have updated the class and variable.